### PR TITLE
vcontrold: fix install script

### PR DESCRIPTION
**Pull request by reitermarkus**

### CODE REVIEW

File: vcontrold.rb [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+      chmod 0755, "auto-build.sh"
 
-    system "./configure", "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-
-    system "make", "install"
+      system "./auto-build.sh"
+      system "./configure",
+             "--prefix=#{prefix}",
+             "--with-xml2-include-dir=/usr/local/opt/libxml2/include/libxml2"
```
Comment (dunn) [inline vcontrold.rb]: Not everyone installs into `/usr/local`, so this should be `--with-xml2-include-dir=#{Formula["libxml2"].opt_include}/libxml2`.

Comment (reitermarkus) [inline vcontrold.rb]: Thanks, I didn't know how to dynamically get the path.

**Comment by reitermarkus** [comment]
I think this could be merged now.

**Comment by reitermarkus** [comment]
Push.

**Comment by dunn** [comment]
Sorry for the delay! Merged in 1a1c222052d091176fed9a51f84309c31a6f32c2, thanks!

**Comment by reitermarkus** [comment]
No problem, thanks.

**Pull request closed without merging**